src/pages/versions.md

```diff
@@ -41,20 +41,31 @@ Below are the documentation and release notes for the currently released version
 
 ### Past versions
 
-Below are the documentation and release notes for previous versions of Rancher 2.7.x:
```

Comment:
I think this line is still needed below to denote the Rancher 2.7.x release items, wasn't sure where to add in below but wanted to make known.

Reply:
Thanks for catching this Sunil. Fixed in 7cb977a7.